.github/workflows/ci.yml

```diff
@@ -50,4 +50,4 @@ jobs:
       - run: rustup override set ${{env.MSRV}}
       - run: rustup component add clippy
       - uses: Swatinem/rust-cache@v2.2.0
-      - run: cargo clippy --workspace -- --deny warnings
+      - run: cargo clippy --workspace -- --deny warnings --allow clippy::suspicious_else_formatting
```

Comment:
maybe we should put it in the lib.rs instead in case people are running clippy locally?

Reply:
I've added a similar attribute in https://github.com/conduit-rust/conduit/pull/100, but no luck (https://github.com/conduit-rust/conduit/actions/runs/3510133627/jobs/5879698025). The weird thing is that this doesn't reproduce on my end (even with 1.56). Maybe bumping up the Rust version on Clippy CI gets rid of the failure.